modules/ROOT/pages/external-identity.adoc

```diff
@@ -49,9 +49,9 @@ The IdP you select is effective for the entire organization and all business gro
 
 For more information, see the documentation for your identity provider.
 
-After configuring identity management, you can't add users to your organization in Anypoint Platform. In Access Management, *Invite User* is disabled to prevent this operation. To add new users your organization, include them in your external identity management solution according to your normal internal provisioning process.
+After configuring identity management, you must add new SSO users using your external identity management solution according to your normal internal provisioning process. If you use Invite User feature to add users to your organization after you have configured an identity provider, the credentials for these users are stored locally in your organization rather than with the identity provider.
```

Comment:
"according to your normal" is strange. What about "...solution and internal provisioning process." ?

Reply:
```suggestion
After configuring identity management, you must add new SSO users using your external identity management solution and internal provisioning process. If you use the Invite User feature to add users to your organization after you have configured an identity provider, the credentials for these users are stored locally in your organization rather than with the identity provider.
```